.github/workflows/merge.yml

```
@@ -70,6 +70,11 @@ jobs:
         run: |
           . venv/bin/activate
           uv build
+      - name: Generate PEP 740 Attestations
+        if: steps.check_package.outputs.should_publish == 'true'
```

actionlint (1.7.12)

[error] property "check_package" is not defined in object type {}

(expression)